src/pages/Login/Login.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import LoginInput from './components/LoginInput';
+import styled from 'styled-components';
+
+function Login() {
+  return (
+    <Container>
+      <article>
+        <Title>
+          <h2>
+            <img alt="cream로고" src="/images/login_logo.png" />
+          </h2>
+        </Title>
+        <form>
+          {loginData.map(loginData => {
+            const { id, type, text, discription, placeholder } = loginData;
+            return (
+              <LoginInput
+                key={id}
+                type={type}
+                text={text}
+                discription={discription}
+                placeholder={placeholder}
+              />
+            );
+          })}
+          <LoginButton>
+            <img src="https://t1.daumcdn.net/cfile/tistory/2207573D58CFDE2704" />
+            <span>카카오 로그인</span>
```

Comment:
```suggestion
            카카오 로그인
```
굳이 span 태그로 감싸지 않고 바로 text를 사용하셔도 됩니다

Reply:
버튼 태그 안의 컨텐츠간 margin을 주고 싶어서 사용해봤는데, `<span>`없이 사용해봐도 되네요 ㅋㅋㅋㅋㅋ.. 감사합니다 ㅎㅎ 수정했습니다!!